submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/security/SubmitPermissionTest.kt

```diff
@@ -184,6 +185,36 @@ class SubmitPermissionTest(
         assertThat(regularUserWebClient.submitSingle(resubmission, SubmissionFormat.TSV)).isSuccessful()
     }
 
+    @Test
+    fun `4-8 owner resubmits without attach permission`() {
```

Comment:
This test is not clear for me, if what we want to test is that "owner resubmits without attach permission" why we are using belhalf client and a imporsonated user in the test.  test structure should be something like

1.  submit with user A in project B
2. Remove Attach permission from user A
3. Re submit and validate success.

Do am I missing something?

Reply:
No, you're not, the test case you're describing is correct. The reason why I added the onbehalf step was to recreate the scenario most used by Ahmed for AE submissions